src/js/hca/components/HealthCareApp.jsx

```diff
@@ -329,19 +332,44 @@ class HealthCareApp extends React.Component {
     //   }
     // }
 
+    // Until we come up with a common code base between this and the schemaform
+    //  forms, the following is borrowed from NavHeader
+    let step;
+    chapters.forEach((chapter, index) => {
+      if (chapter.pages.some(page => page.path === this.props.location.pathname)) {
+        step = index + 1;
+      }
+    });
+
+    let contentClass = classNames(
+      'progress-box',
+      'progress-box-schemaform',
+      // Align the intro and confirmation content with the title
+      { 'intro-content': _.includes(['/introduction', '/submit-message'], this.props.location.pathname) }
+    );
+
     return (
       <div>
         <div className="row">
           <Element name="topScrollElement"/>
+          {/*
           <div className="medium-4 columns show-for-medium-up">
             <Nav
                 data={this.props.data}
                 pages={this.props.uiState.sections}
                 chapters={chapters}
                 currentUrl={this.props.location.pathname}/>
           </div>
+          */}
           <div className="medium-8 columns">
-            <div className="progress-box">
+            <FormTitle title="Apply online for health care with the 10-10ez" subTitle="OMB No. 2900-0091"/>
```

Comment:
Is this block exactly the same as on the 1990 form? We could pull it into a separate component if so.